### PR TITLE
 feat(heade): add header component

### PR DESCRIPTION
**Pull request by CaileanSong**
add 2023-12-09 at 22 44 17

### CODE REVIEW

File: components/header/tag/tag.vue
Comment (elonehoo) [inline components/header/tag/tag.vue:1]: what is `text-ms`, are you trying to write `text-sm` ?

Comment (CaileanSong) [reply components/header/tag/tag.vue:1]: yes,I think I typed it wrong

**Comment by elonehoo** [comment]
it doesn't look like it's finished.

**Comment by elonehoo** [comment]
when the sidebar is revenue, the page error

image

**Comment by elonehoo** [comment]
After:

image

**Comment by elonehoo** [comment]
i believe that so far, the style and basic groundwork have been completed. Next, we need to incorporate some of the business functions.

cc @CaileanSong

**Comment by elonehoo** [comment]
i think the branch is ready for review.

@CaileanSong what do you think?

**Review by elonehoo — APPROVED** [review]

**Pull request merged**